scripts/install_deps_mac.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -x # echo on
+set -e # exit on error
+
+brew install --verbose \
+    assimp \
+    boost \
```

Comment:
since we build boost i recommend deleting it from this list. as well as the other deps.